### ISSUE TITLE
Path for minified script is incorrect

### ISSUE THREAD

**Issue by onetrev** [xwp/wp-dependency-minification#72]
For some reason the plugin is generating a path that is missing the / at the start of the path for the concatenated files. As such, no styles or js load since the generate URL does not exist.

The plugin generates this path:

`http://testwp.localhost_minify/.......`

whereas there should be a / in front of the **_minify** like so:

`http://testwp.localhost/_minify/.....`

Am I missing something super obvious here, or is this a bug?

Details:
WP 4.1.1
Apache 2.4.9 / PHP 5.5.12

**Comment by utsavdevsharma** [comment]
Yes same issue , is there a solution or config ? @onetrev @westonruter @shadyvb ??
